### PR TITLE
Small edits for PypeIt compatability

### PR DESCRIPTION
**Pull request by profxj**
As titled.

No new tests or docs

### CODE REVIEW

**Comment by profxj** [comment]
Also has a bug fix for the stack plot test (the last PR tripped it and
I didn't notice).

File: linetools/spectralline.py [hunk 1]
```diff
@@ -28,6 +28,7 @@
 zero_coord = SkyCoord(ra=0.*u.deg, dec=0.*u.deg)  # Coords
 init_analy = {
             'spec': None,              # Analysis inputs (e.g. spectrum; from .clm file or AbsID)
+            'spec_file': '',           # Spectrum file
```
Comment (jnburchett) [inline linetools/spectralline.py]: This wasn't there already?

Comment (profxj) [reply linetools/spectralline.py]: oddly, no.

**Comment by profxj** [comment]
merging.

**Pull request merged**